src/voice_generator/gui.py

```
@@ -0,0 +1,128 @@
+    Slider,
+    Markdown,
+    Accordion,
+    Row,
+    Column,
+    TabbedInterface,
+)
+from voice_generator import (
+    CUDA_AVAILABLE,
+    CHAR_LIMIT,
+    STREAM_NOTE,
+    TOKEN_NOTE,
```

Ruff (0.8.2)

`voice_generator.TOKEN_NOTE` imported but unused

Remove unused import: `voice_generator.TOKEN_NOTE`

(F401)